sdk/vision/azure-ai-vision-imageanalysis/README.md

```diff
@@ -0,0 +1,337 @@
+# Azure Image Analysis client library for Python
+
+The Image Analysis service provides AI algorithms for processing images and returning information about their content. In a single service call, you can extract one or more visual features from the image simultaneously, including getting a caption for the image, extracting text shown in the image (OCR) and detecting objects. For more information on the service and the supported visual features, see [Image Analysis overview](https://learn.microsoft.com/azure/ai-services/computer-vision/overview-image-analysis?tabs=4-0), and the [Concepts](https://learn.microsoft.com/azure/ai-services/computer-vision/concept-tag-images-40) page.
+
+Use the Image Analysis client library to:
+* Authenticate against the service
+* Set what features you would like to extract
+* Upload an image for analysis, or send an image URL
+* Get the analysis result
+
+[Product documentation](https://learn.microsoft.com/azure/ai-services/computer-vision/overview-image-analysis?tabs=4-0) 
+| [Samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/vision/azure-ai-vision-imageanalysis/samples)
+| [Vision Studio](https://portal.vision.cognitive.azure.com/gallery/imageanalysis)
+| [API reference documentation](https://learn.microsoft.com/python/api/azure-ai-vision) <!-- TODO: replace with https://learn.microsoft.com/python/api/azure-ai-vision-imageanalysis after ref docs are published -->
+| [Package (Pypi)](https://pypi.org/project/azure-ai-vision) <!-- TODO: replace with https://pypi.org/project/azure-ai-vision-imageanalysis/ after package ships -->
+| [Package (Conda)](https://anaconda.org/microsoft/azure-ai-vision-imageanalysis)
+| [SDK source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/vision/azure-ai-vision-imageanalysis/azure/ai/vision/imageanalysis)
+
+## Getting started
+
+### Prerequisites
+
+* [Python 3.7](https://www.python.org/) or later installed, including [pip](https://pip.pypa.io/en/stable/).
```

Comment:
Given you plan to ship early next year, please use Python 3.8. :)

Reply:
Updated to 3.8. Thanks @xiangyan99 !